source/TimePicker.js

```diff
@@ -455,9 +457,10 @@
 						]},
 						this
 					);
+					pickers.push(createdComponent.getClientControls()[0]);
```

Comment:
I see why you need to do this from a logistical perspective. Going back to the change where the wrapper control around the picker was added (i.e. with the `moon-date-picker-wrap`) class, `this.pickers` has been an array consisting of wrapper controls, rather than the actual hour or minute picker, so it has been incorrect for a while. I think we will need to do something similar for `DatePicker` as well, right?